zabbix/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "display_name": "zabbix",
+  "maintainer": "KosukeKamiya@users.noreply.github.com",
+  "manifest_version": "1.0.0",
+  "name": "zabbix",
+  "metric_prefix": "zabbix.",
+  "metric_to_check": "",
```

Comment:
Are there any default metrics that are collected? `metric_to_check` allows the integration tile to get marked as installed when successfully configured.

Reply:
There is no default metrics. 

This integration only collects the metrics defined in conf.yaml.